src/utils/formatting/content_format.rs

```diff
@@ -123,13 +124,17 @@ pub fn format_row(
             .iter()
             .map(|line| align_line(line.to_string(), info, cell));
 
-        // Style the cell if necessary.
+        // Style the cell if necessary
+        #[cfg(feature = "tty")]
         if table.should_style() {
             let cell_lines = cell_lines.map(|line| style_line(line, cell));
             temp_row_content.push(cell_lines.collect());
         } else {
             temp_row_content.push(cell_lines.collect());
         }
+
+        #[cfg(not(feature = "tty"))]
+        temp_row_content.push(cell_lines.collect());
```

Comment:
I'm not exactly sure, but something bothers me about these inline feature flags.
I usually try to separate the different code branches on a function or module level in my platform-/target specific code.

This is probably very subjective, but at least for me it makes understanding the code flow much easier.

What do you think about this:
```
        // Apply tty styling for this cell.
        #[cfg(feature = "tty")]
        let cell_lines = apply_tty_styling(table, cell, cell_lines).into_iter();

        temp_row_content.push(cell_lines.collect());
    }

    #[cfg(feature = "tty")]
    /// A small wrapper around the top-level cell styling logic. It's only used to have a clear
    /// separation of our tty styling logic for the `tty` feature flag.
    fn apply_tty_styling(
        table: &Table,
        cell: &Cell,
        cell_lines: impl Iterator<Item = String>,
    ) -> Vec<String> {
        if table.should_style() {
            let cell_lines = cell_lines.map(|line| style_line(line, cell));
            cell_lines.collect()
        } else {
            cell_lines.collect()
        }
    }
```

I guess its much more verbose, but the separation should be clearer.